### PR TITLE
Fix: Add netloc to redirect after OIDC auth

### PR DESCRIPTION
**Pull request by tomchop**

### CODE REVIEW

File: yeti.conf.sample [hunk 1]
```diff
@@ -8,6 +8,11 @@ export_root = /var/opt/yeti
 logging = /var/log/yeti_user_activity.log
 plugins_path = ./plugins
 
+# Public schee + hostname + port for Yeti. Use it if you want to specify an
```
Comment (udgover) [inline yeti.conf.sample:11]: typo: s/schee/scheme

**Review by udgover — CHANGES_REQUESTED** [review]
I assume you were doing tests in real environment and had to setup the relevant redirect uri so it's LGTM, just requesting a small change for a typo in config doc.

File: yeti.conf.sample (hunk 1 above)
Comment (tomchop) [inline yeti.conf.sample:11]:
```suggestion
# Public scheme + hostname + port for Yeti. Use it if you want to specify an
```

**Review by udgover — COMMENTED** [review]
LGTM!

**Pull request merged**